docs/manual/theme-manager/manage-template.de.md

```diff
@@ -0,0 +1,135 @@
+verwendet und beinhaltet HTML und PHP Code. Unter dem Menüpunkt `Layout\Templates` können die Dateien erstellt, 
+in Ordnern abgelegt und bearbeitet werden. Diese Anpassungen sind updatesicher.
+
+{{% notice note %}}
+Im [Debug-Modus](../../system/debug-modus/) werden die Template Namen im HTML Quellcode als Kommentare ausgegeben. 
+Man kann hierüber nachvollziehen welches Template zum Einsatz kommt.
+{{% /notice %}}
+
+
+## Ordner Struktur
+
+Ein Template kann im `Layout\Templates` Hauptverzeichnis abgelegt werden. Das entsprechende Template wird dann z. B. 
+in einem Inhaltselement zur Auswahl angeboten und als `global` gekennzeichnet.
+
+Im [Theme-Manager](../../theme-manager/themes-verwalten) kann man einen vorhandenen Template Ordner mit dem Theme 
+verknüpfen. Die Template Dateien in diesem Ordner werden bei der Auswahl dann mit dem jeweiligen `Theme-Namen` gekennzeichnet.
+
+{{% notice note %}}
+Template Dateien in weiteren Unterodnern werden bei der Auswahl nicht berücksichtigt.
+{{% /notice %}}
+
+
+## Dateinamen
+
+Die Template Dateien werden zur Erkennung mit einem Präfix versehen. Beispielsweise deutet `ce_` auf ein
+Inhaltselement hin. Möchte man z. B. die Ausgabe des Inhaltselements vom Typ Text ändern kann man das 
+Template `ce_text.html5` hierzu verwenden. 
+
+In diesem Fall haben die Template Änderungen Auswirkung auf alle Inhaltselemente vom Typ Text. Dies ist nicht immer
+erwünscht. Zur gezielten Nutzung kann man das Template individuell bezeichnen. Hierbei muss die jeweils vorgegebene
+Template Bezeichnung beibehalten und lediglich erweitert werden. Also z. B. `ce_text.html5` 
+umbenennen nach  `ce_text_individuell.html5`.
+
+Dieses Template kann dann gezielt zur Ausgabe für ein (o. mehrere) Inhaltselement(e) vom Typ Text genutzt werden.
+
+
+## CSS und JavaScript Assets
+
+Oft werden zu einem individuellen Template zusätzliche Assets wie CSS oder JavaScript Dateien benötigt. Man kann diese 
+Dateien grundsätzlich über das Seitenlayout eines Themes einbinden. Allerdings werden die Assets dann immer geladen 
+auch wenn diese auf verschiedenen Seiten gar nicht benötigt werden. Es ist daher sinnvoll diese Dateien im Template 
+selbst anzugeben. Hierbei stehen verschiedene Möglichkeiten zur Verfügung. 
+
+Am einfachsten ist es die Dateien im Contao `files` Verzeichnis anzulegen und dann im Template zu referenzieren:
+
+```php
+<link href="files/custom.css" rel="stylesheet">
+<script src="files/custom.js"></script>
+```
+
+Alternativ kann man die Assets im Template auch so hinterlegen, das diese z. B. im Html Header der Seite ausgegeben
+werden:
+
+```php
+$GLOBALS['TL_CSS'][] = 'files/custom.css|static';
+$GLOBALS['TL_JAVASCRIPT'][] = 'files/custom.js|static';
+```
+
+Diese Umsettzung bietet weitere Optionen. Mit Angabe von `static` werden die Dateien z. B. zu den bestehenden Assets
+eines Seitenlayouts hinzugefügt bzw. diese zusammengefasst. Eine detaillierte Beschreibung aller Optionen findest Du hier:
```

Comment:
findest du in der Entwickler-Dokumentation unter [Adding CSS & JavaScript Assets](https://docs.contao.org/dev/framework/asset-management/).

Personalpronomen schreiben wir klein.